File: src/14/14004.js
/**
 * 14004. ICPC
 * 
 * 작성자: xCrypt0r
 * 언어: node.js
 * 사용 메모리: 8,328 KB
 * 소요 시간: 124 ms
 * 해결 날짜: 2020년 12월 4일
 */

const fs = require('fs');

function main() {
    let [A, B, C, D] = fs.readFileSync('/dev/stdin').toString().trim().split(' ').map(x => +x);

    console.log(Math.floor((C + D) / (A - B)));
}

main();

File: src/14/14004.py
"""
14004. ICPC

작성자: xCrypt0r
언어: Python 3
사용 메모리: 29,088 KB
소요 시간: 68 ms
해결 날짜: 2020년 12월 4일
"""

def main():
    A, B, C, D = map(int, input().split())

    print((C + D) // (A - B))

if __name__ == '__main__':
    main()